src/vanilla.ts

```diff
@@ -4,6 +4,9 @@ const VERSION = Symbol()
 const LISTENERS = Symbol()
 const SNAPSHOT = Symbol()
 
+const isProduction =
+  typeof process === 'object' && process.env.NODE_ENV === 'production'
```

Comment:
```suggestion
  typeof process !== 'object' || process.env.NODE_ENV === 'production'
```
You failed to invert the logic. 😄

Reply:
Yea, I forgot. thanks.